libavcodec/x86/vvc/vvcdsp_init.c

```diff
@@ -204,6 +204,18 @@ AVG_FUNCS(16, 12, avx2)
     c->inter.avg    = bf(avg, bd, opt);                                 \
     c->inter.w_avg  = bf(w_avg, bd, opt);                               \
 } while (0)
+
+int ff_vvc_sad_8_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
+int ff_vvc_sad_16_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
+int ff_vvc_sad_32_128_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
+
+#define SAD_INIT() do {                                           \
+    c->inter.sad[1] = ff_vvc_sad_8_avx2;                          \
```

Comment:
why ff_vvc_sad_4_avx2 is not needed?

Reply:
For DMVR it's only valid for widths >= 8, height >= 8

Reply:
![image](https://github.com/ffvvc/FFmpeg/assets/15571478/048d873f-acca-49c3-b7ec-0221581a1e10)
https://ieeexplore.ieee.org/document/9252910/